### DISCUSSION TITLE
PARTE DEL PASTELERO

### DISCUSSION THREAD

**Discussion by AlfredoLandero** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#251] [Q&A]
Buenas noches. creo que alguien ya había preguntado algo similar pero aun no me queda claro. La parte del pastelero debe ser una pagina nueva?. La pagina del cliente debe enviar los datos a la del pastelero?. o como debe funcionar esa parte?. De antemano gracias por sus respuestas.

**Comment by alanglez45** [comment]
Pay attention to the mission commander bro.  

Pastelero y pastelería son páginas independientes.

Sólo son páginas frontend. Para enviar y recibir información se requiere del backend (cosa que aun no hemos visto)

**Comment by JuanSalvador73** [comment]
Son 2 páginas por separado, una es la de la pastelería, donde viene la info de los pasteles y la que tiene un formulario para hacer pedidos, la otra es la que recibe el pastelero ya con los datos que el cliente le envió de su pedido, como tipo de pastel, sabor, ornamentos, número de pedidos, etc. Además hay que incluir un logo de tu pastelería. Puede ser algo muy sencillo. 

Básicamente la 2 página es una tabla. No hay nada interactivo aún, solo el puro maquetado en HTML. La parte de programación supongo que la veremos mas adelante. Es cuando llenas el formulario con tus datos y el pastelero (admin) recibe esa info.

Varios explorers han publicado su trabajo, échales un ojo. Por cierto este encargo NO necesariamente tiene que tener ya estilos CSS integrados - aunque ello si es obviamente un plus -  solo el HTML por ahora. Pero entre mas completo lo tengas mejor.

**Reply by AlfredoLandero** [reply]
Muchas gracias por tu respuesta.  Dudas aclaradas.
